help/follow-a-topic.md

```diff
@@ -112,6 +118,19 @@ messages just in followed topics, without counting other channel messages.
 
 {!manage-configured-topics-desktop-web.md!}
 
+{tab|mobile}
+
+1. Tap the **Channels** (<i class="zulip-icon zulip-icon-hash mobile-help"></i>)
+   tab at the bottom of the app.
+
+1. Select a channel containing topics you want to configure.
+
+1. Tap **Topics** in the upper right corner of the app.
```

Comment:
We plan to replace the "TOPICS" text button with an icon: https://github.com/zulip/zulip-flutter/issues/1532 I'm OK with this commit being merged as-is, and I'll make a note on the zulip-flutter issue to update this doc.

Also nit: the current button "TOPICS" is in all caps.

Reply:
Our pattern for all-caps font has been to use all-lowercase in the help center. E.g., "views" in https://zulip.com/help/left-sidebar.